pkgs/applications/networking/mullvad/default.nix

```
@@ -0,0 +1,117 @@
+{ stdenv, makeWrapper, fetchurl, dpkg
+, alsaLib, atk, cairo, cups, dbus, expat, fontconfig, freetype
+, gdk_pixbuf, glib, gnome2, nspr, nss, gtk3, gtk2, at-spi2-atk
+, gsettings-desktop-schemas, gobject-introspection, wrapGAppsHook
+, xorg, nghttp2
+, libudev0-shim, glibc, curl, openssl, autoPatchelfHook
+, udev, pulseaudio, libpulseaudio, systemd, libnotify
+}:
+
+let deps = [
+    alsaLib
+    at-spi2-atk
+    atk
+    cairo
+    cups
+    dbus
+    expat
+    fontconfig
+    freetype
+    gdk_pixbuf
+    glib
+    gnome2.GConf
+    gnome2.pango
+    gtk2
+    gtk3
+    gsettings-desktop-schemas
+    pulseaudio
+    libpulseaudio
+    libnotify
+    xorg.libX11
+    xorg.libXScrnSaver
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXtst
+    xorg.libxcb
+    nspr
+    nss
+    udev
+    stdenv.cc.cc
+    systemd
+  ];
+
+in
+
+stdenv.mkDerivation rec {
+  name = "mullvad-${version}";
```

```suggestion
  pname = "mullvad";
```

When `pname` and `version` attributes are present they get concatenated like `name = "${pname}-${version}"` automatically.